actix-files/src/named.rs

```diff
@@ -133,6 +133,7 @@ impl NamedFile {
                 mime::APPLICATION => match ct.subtype() {
                     mime::JAVASCRIPT | mime::JSON => DispositionType::Inline,
                     name if name == "wasm" => DispositionType::Inline,
+                    name if name == "xhtml" => DispositionType::Inline,
```

Comment:
We could shorten like:

```suggestion
                    name if name == "wasm" || name == "xhtml" => DispositionType::Inline,
```